plenum/test/test_node.py

```diff
@@ -379,7 +379,7 @@ def __init__(self,
         self.pluginPaths = pluginPaths
 
         self.testNodeClass = testNodeClass
-        self.nodes = OrderedDict()  # type: Dict[str, TestNode]
+        self.nodes = SortedDict()  # type: Dict[str, TestNode]
```

Comment:
Why this change, `self.nodes` is meant to preserve order of insertion, not alphabetical order of names